Makefile

```diff
@@ -551,10 +551,7 @@ docker-auth:
 	fi;
 
 # Docker isolated rampart
-docker-isolated:
-	TAG=$(COMPOSE_TAG) DEV_DOCKER_TAG_BASE=$(DEV_DOCKER_TAG_BASE) docker-compose -f tools/docker-compose.yml -f tools/docker-isolated-override.yml create
-	docker start tools_awx_1
-	docker start tools_isolated_1
+docker-compose-isolated:
```

Comment:
this looks fine to me

there were previously steps between the create and up steps, but looks like they were removed at some point.

might want someone else to review the volume mapping items